.github/workflows/deployment.yml

```diff
@@ -11,7 +13,27 @@ on:
 
 jobs:
 
-  build:
+  Version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/setup-git-token@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create tag if necessary
+        if: github.event_name == 'schedule'
+        run: |
+          git fetch --tags
+          if [[ $(git tag -l --points-at HEAD) = '' ]]; then
+            echo '::set-output name=created::yes'
+            export VER=$(npx daily-version)
+            git tag $VER -m $VER
+            git push origin $VER
+          fi
```

Comment:
Maybe we should add an `else` clause here and `echo` that there are no new changes to publish, hence skipping.

Can do the same if we have successfully created a tag.

Reply:
Sounds good. I hope to find the time to wrap this into an action so we reduce the amount of code to copy/maintain in all repos.